projects/components/src/filtering/filter/filter-operators.ts

```diff
@@ -9,6 +9,7 @@ export const enum FilterOperator {
   GreaterThanOrEqualTo = '>=',
   Like = '~',
   In = 'IN',
+  Contains = 'CONTAINS',
```

Comment:
recommendation : would be helpful to add couple lines describing the differences (or use cases) between LIKE and CONTAINS

Reply:
added